composer.json

```diff
@@ -40,7 +40,7 @@
   "require-dev": {
     "phpunit/phpunit": "^9",
     "mockery/mockery": "^1",
-    "squizlabs/php_codesniffer": "^3",
+    "squizlabs/php_codesniffer": "^3.6 || 3.x-dev",
```

Comment:
There's [a bug in current releases of the linter][1] regarding PHP 8 constructor property promotion: this works around it by pinning to head of development. Will remove this when next release available.

[1]:https://github.com/rollbar/rollbar-php/pull/517/commits/5e147d239eb274249f014d01f59da63310d2e305